### DISCUSSION TITLE
Request headers are not printed even with "debug" logs enabled

### DISCUSSION THREAD

**Discussion by nethi** [encode/httpcore#935] [Q&A]
I followed enabling debug logging instructions at https://www.python-httpx.org/logging/

For the request, I only see log like this and no headers printed

DEBUG [2024-07-13 11:07:17] httpcore.http11 - send_request_headers.started request=<Request [b'POST']>
2024-07-13 11:07:17 - send_request_headers.started request=<Request [b'POST']>
DEBUG [2024-07-13 11:07:17] httpcore.http11 - send_request_headers.complete
2024-07-13 11:07:17 - send_request_headers.complete

Looking at the code, I see  that request object's __repr__ is only printing the method, as logs indicate.

https://github.com/encode/httpcore/blob/acf7d151c715d0b18870ecc660a8677f57f0bcea/httpcore/_sync/http11.py#L87C1-L96C31
```python
        try:
            kwargs = {"request": request}
            try:
                with Trace(
                    "send_request_headers", logger, request, kwargs
                ) as trace:
                    self._send_request_headers(**kwargs)
                with Trace("send_request_body", logger, request, kwargs) as trace:
                    self._send_request_body(**kwargs)
            except WriteError:
```
            
https://github.com/encode/httpcore/blob/acf7d151c715d0b18870ecc660a8677f57f0bcea/httpcore/_models.py#L364
```python
    def __repr__(self) -> str:
        return f"<{self.__class__.__name__} [{self.method!r}]>"
```
Simple modification seems to take care of printing headers.  Is there any way to achieve this or should I raise a feature request ?

```python
    def __repr__(self) -> str:
        return f"<{self.__class__.__name__} [{self.method!r}][{self.headers}]>"
```

**Comment by lovelydinosaur** [comment]
I think we should probably just keep it concise.
Similarly, we don't include the body in the `send_request_body` logging.
